core/src/main/java/com/orange/ouds/core/component/OudsCheckbox.kt

```diff
@@ -116,7 +120,8 @@ fun OudsCheckbox(
  * and relies entirely on a higher-level component to control the state.
  * @param modifier [Modifier] applied to the layout of the checkbox.
  * @param enabled Controls the enabled state of the checkbox. When `false`, this checkbox will not be clickable.
- * @param error Controls the error state of the checkbox.
+ * @param error Optional [OudsError] to provide in the case of the checkbox should appear in error state, `null` otherwise.
+ * vocalized by screen readers.
```

Comment:
This sentence does not appear in the documentation of `OudsCheckbox`. Should we keep it?